rust_src/crates/webrender/src/wrterm.rs

```diff
@@ -221,8 +219,8 @@ pub extern "C" fn image_pixmap_draw_cross(
 /// Hide the current tooltip window, if there is any.
 /// Value is t if tooltip was open, nil otherwise.
 #[lisp_fn]
-pub fn x_hide_tip() -> bool {
-    false
+pub fn x_hide_tip() -> LispObject {
```

Comment:
does those bools cause problems to you?
I think `lisp_fn` converts them into Qt/Qnils automatically.